astrosat_users/adapters.py

```diff
@@ -294,6 +294,14 @@ def send_password_confirmation_email(self, user, email, **kwargs):
 
         self.send_mail(template_prefix, email, context)
 
+    def send_mail(self, template_prefix, email, context, **kwargs):
+        cc = kwargs.pop("cc", [])
+        bcc = kwargs.pop("bcc", [])
+        msg = self.render_mail(template_prefix, email, context)
+        msg.cc = list(cc)
+        msg.bcc = list(bcc)
```

Comment:
If not using an array only, should this not use a `set`, to ensure there are no duplicates? I don't see what the list gives us over a basic array here?

Reply:
You've caught an error, but not for the reason you think.

I called `list` explicitly in-case "cc" or "bcc" is passed in as a string.  However, 

```
>>> x = "foo@bar.com"
>>> list(x)
['f', 'o', 'o', '@', 'b', 'a', 'r', '.', 'c', 'o', 'm']
```

Clearly, that's not what I want to happen.  I will make another commit to fix this.

Reply:
That is what I meant, just didn't word it right.... and if you believe that, can I interest you in an email I have from an African Prince.

Reply:
I still want to use `list` though b/c I generally pass a queryset to this fn which needs to be converted into a simple list for the django email stuff to work.

Reply:
fair enough, sounds sensible.